src/autocomplete.js

```diff
@@ -489,33 +591,23 @@ class Autocomplete extends Component {
     // if they don't then we need to clone the element they return and
     // apply the props for them.
     this.getRootProps.called = false
-    const {
-      children,
-      // eslint-disable-next-line no-unused-vars
-      defaultValue,
-      // eslint-disable-next-line no-unused-vars
-      getValue,
-      // eslint-disable-next-line no-unused-vars
-      getA11yStatusMessage,
-      // eslint-disable-next-line no-unused-vars
-      defaultHighlightedIndex,
-      // eslint-disable-next-line no-unused-vars
-      multiple,
-      // eslint-disable-next-line no-unused-vars
-      onClick,
-      // eslint-disable-next-line no-unused-vars
-      onChange,
-      ...rest
```

Comment:
Decided we don't need to forward props. Because you're rendering the child component/element yourself, you can apply the props you want directly to that element.